pkg/api/image.go

```diff
@@ -86,6 +87,35 @@ func (h *Handler) GetImages(w http.ResponseWriter, r *http.Request) {
 	FormatJsonResponse(w, result)
 }
 
+// Creates a new file system in this cluster.
+// POST
+// /file
+func (h *Handler) CreateFileSystem(w http.ResponseWriter, r *http.Request) {
+
+	if err := file.EnableFileSystem(h.context); err != nil {
+		logger.Errorf("failed to create file system: %+v", err)
+		w.WriteHeader(http.StatusInternalServerError)
```

Comment:
there are more cases we need to check for, such as if the file system already exists, if we're out of space, or if the cluster isn't available yet, etc. We can return a better http error for those, but for the basic case implemented thus far it's really just a generic and unexpected error